app/mailers/user_mailer.rb

```diff
@@ -38,6 +38,16 @@ def add_reviewer(reviewer_id, paper_id)
       subject: "You've been added as a reviewer on Tahi")
   end
 
+  def assigned_editor(editor_id, paper_id)
```

Comment:
Unused method argument - `editor_id`. If it's necessary, use `_` or `_editor_id` as an argument name to indicate that it won't be used.